package.json

```diff
@@ -9,12 +9,14 @@
     "stream": "readable-stream",
     "http": "stream-http",
     "ipfs-api": false,
-    "ipfs": false
+    "ipfs": false,
+    "./node_modules/ipfsd-ctl/src/ipfsd-in-proc.js": false,
+    "./node_modules/ipfsd-ctl/src/factory-in-proc.js": false
   },
   "scripts": {
     "test": "aegir test",
     "test:node": "aegir test -t node",
-    "test:browser": "aegir test -t browser",
+    "test:browser": "node --max-old-space-size=8192 ./node_modules/.bin/aegir test -t browser",
```

Comment:
Still no luck. Kind of running out of my options here :(

Reply:
@vmx @pgte @dignifiedquire any Webpack wisdom to solve this issue?

Reply:
Actually, spoke too soon, IT RUUUNS :D 

<img width="627" alt="image" src="https://user-images.githubusercontent.com/1211152/36060145-3829cbf4-0e3b-11e8-9a53-e3cc54736da0.png">